lib/Star/Kata/Command/StartCommand.php

```diff
@@ -60,18 +61,11 @@ public function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        try {
-            $kataName = $input->getArgument('kata');
-            if (empty($kataName)) {
-                throw new \RuntimeException('Kata must be supplied.');
-            }
+        $kataName = $input->getArgument('kata');
 
-            $kata = $this->collection->getKata($kataName);
-            if (null === $kata) {
-                throw new \RuntimeException("The '{$kataName}' kata was not found.");
-            }
+        try {
+            $kata = $this->service->startKata($kataName);
```

Comment:
is this a read or write operation? Are you separating your commands from your queries? Given the nature of this command, I would expect a start kata command to be sent to the `KataService`, and then a subsequent query operation.